etna/commands/forecast_command.py

```diff
@@ -83,9 +120,17 @@ def forecast(
 
     tsdataset = TSDataset(df=df_timeseries, freq=freq, df_exog=df_exog, known_future=k_f)
 
+    horizon: int = pipeline_configs["horizon"]  # type: ignore
+    horizon = compute_horizon(horizon=horizon, forecast_params=forecast_params, tsdataset=tsdataset)
+    pipeline_configs["horizon"] = horizon  # type: ignore
+
+    forecast_args = drop_additional_forecast_params(forecast_params)
```

Comment:
It a difficult to understand naming. May be it is better to name it `forecast_call_params` or `forecast_method_params`.

Reply:
Sorry, I meant variable `forecast_args`, not method name.

Reply:
Oh, ok)